queries.sql

```diff
@@ -31,3 +31,56 @@ WHERE neutered = true;
 SELECT *
 FROM animals
 WHERE  weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+-- Transactions and modifications
+
+BEGIN;
+UPDATE animals SET species = 'undefined';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE animal_name LIKE '%mon';
+
+UPDATE animals SET species = 'pokemon' WHERE animal_name IN ('Charmander', 'Squirtle', 'Blossom', 'Ditto', 'Pikachu');
+COMMIT;
```

Comment:
You should verify that the data was persisted with a `SELECT` query before and after this `COMMIT`.